packages/shorebird_cli/lib/src/executables/patch_executable.dart

```diff
@@ -49,9 +50,20 @@ class PatchExecutable {
     final result = await process.run(diffExecutable, diffArguments);
 
     if (result.exitCode != ExitCode.success.code) {
+      var messageDetails = '';
+
+      if (result.exitCode < 0 && platform.isWindows) {
```

Comment:
There is a specific error code which means this is missing. I'm not sure it true in the general negative case that installing that package would help.